api/v2/models/queries.js

```diff
@@ -8,10 +8,12 @@ const createTableUsers = `CREATE TABLE IF NOT EXISTS users(
     address VARCHAR NOT NULL,
     isadmin BOOLEAN NOT NULL,
     password VARCHAR NOT NULL,
-    profilePic VARCHAR NULL
+    profilePic VARCHAR NULL,
+    createdon timestamp default now()
     )`;
 const createTableProperties = `CREATE TABLE IF NOT EXISTS properties(
       id serial PRIMARY KEY,
+      owner INTEGER,
```

Comment:
Make foreign key reference here to the user table

Reply:
make it cascade on delete.

Reply:
I received the feedback and already worked on it.